bdb/llmeta.c

```diff
@@ -2655,6 +2655,11 @@ int bdb_add_dummy_llmeta(void)
     }
 
 retry:
```

Comment:
This addresses another locked-startup condition.  The calling code is holding the bdb-lock in read-mode, while a different thread is attempting to upgrade.  The subsequent wait-for-seqnum-from-all code would wait until MIN_REP_TIMEOUTMS as none of the nodes has a valid coherency lease.  An additional check inside of wait-for-seqnum closes the race here.  The lock could be desired AFTER this check- but that's okay, because we catch it inside of the first part of wait-for-seqnum.  Dorin and I might have similar code pr's for the 'inside wait-for-seqnum' case.